### DISCUSSION TITLE
[Suggestion] Fixed Mode Hide Chat

### DISCUSSION THREAD

**Discussion by oAwks** [runelite/runelite#15945] [Q&A] [answered]
Is it possible for someone to update the plugin: Fixed Mode Hide Chat (plugin hub) to not hide chat when clicking the esc key? 

I really like this plugin but its annoying to use because it acts weird with the default Runescape setting: Esc closes the current interface. I personally use the Esc key to close most interfaces on rs (bound to my mouse) it speeds up banking a ton and saves having to move the mouse to the close interface button. How the plugin works atm is if I click esc it'll minimise the chat box then I have to click esc again to close a interface, It would be nice if it ignored the esc key click or was remappable to another key.

Would greatly appreciate it if someone with the knowledge to make this a reality could.

**Comment by Nightfirecat** [comment, marked as answer]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.
